homeassistant/components/energy/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""The Energy integration."""
+from __future__ import annotations
+
+from homeassistant.components import frontend
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import ConfigType
+
+from . import websocket_api
+from .const import DOMAIN
+
+STORAGE_VERSION = 1
+STORAGE_KEY = DOMAIN
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up Energy."""
+    websocket_api.async_setup(hass)
+    frontend.async_register_built_in_panel(hass, DOMAIN, DOMAIN, "hass:flash")  # type: ignore
```

Comment:
I've typed the frontend here:
https://github.com/home-assistant/core/pull/52148

Reply:
```suggestion
    frontend.async_register_built_in_panel(hass, DOMAIN, DOMAIN, "hass:flash")
```